src/main/java/frc/robot/RobotContainer.java

```diff
@@ -63,9 +72,12 @@ public class RobotContainer {
 
   private final Drive drive;
   private final Vision vision;
-  private final IntakeSubsystem intake = new IntakeSubsystem();
   private final Elevator elevator = new Elevator();
   private final CoralScorer coralScorer = new CoralScorer();
+  public static final Sensors sensors = new Sensors();
```

Comment:
Instead of making this public, can you pass it into the IntakeSubsystem as a new parameter